src/shogun/classifier/GaussianProcessClassification.cpp

```diff
@@ -133,6 +133,8 @@ CBinaryLabels* CGaussianProcessClassification::apply_binary(
 		SG_REF(data);
 
 	CBinaryLabels* result=new CBinaryLabels(get_mean_vector(data));
+	if (m_compute_variance)
+		result->put("current_values", get_variance_vector(data));
```

Comment:
I guess the compiler can figure template parameter here?

Reply:
no I mean: if you would have a typo in `"current_values"` this would lead to a runtime error (bad). So why not use a typed setter or the current values or modify them otherwise using the Dense/Binary Labels API that is available here?

Reply:
are those setters gonna be removed in the future?

Reply:
unclear :) You can also merge it as it is... no big change later